README.md

```diff
@@ -90,7 +87,27 @@ For example:
     
     $ docker run -it docker.io/openshift/jenkins-2-centos7:latest /etc/alternatives/java -jar /usr/lib/jenkins/jenkins.war --version
 
-OpenShift v4 also removes the 32 bit JVM option.  Only 64 bit will be provided for all images.
+Installation (OpenShift V4)
+---------------------------------
+
+    Starting with v4.0, the images are no longer hosted on DockerHub, but instead on Quay.  Their pull specs are:
+    * quay.io/openshift/origin-jenkins:v4.0
+    * quay.io/openshift/origin-jenkins-agent-nodejs:v4.0
+    * quay.io/openshift/origin-jenkins-agent-maven:v4.0
+    * quay.io/openshift/origin-jenkins-agent-base:v4.0
+
+    The images are also still available at the Red Hat Container Catlog.  Replace `openshift3` in the references above with `openshift4`.
```

Comment:
catalog

and the container catalog images are(will be) ose-jenkins-*, not origin, right?

Reply:
I thought the ose-* ones would only show up in the TBR.  That said, I should list those as well.

Will make some updates in the next push.

Reply:
the TBR is the redhat container catalog.  I assumed that's what you intended to refer to.

The available locations are/will be:

quay.io/openshift/origin-jenkins:v4.0  (community)
registry.redhat.io/openshift4/ose-jenknis:v4.0 (ocp) (I think).

though since they're going to have the same UBI base image i'm not even sure what meaningful difference there will be.

I think maybe we are talking about different things though.  Might be better to just specify the actual urls so there's no room for confusion.

Reply:
(also there's still a typo in Catalog above)